content.js

```diff
@@ -70,12 +72,11 @@ async function generateHTML(assets) {
 }
 
 async function addToDOM(html) {
-  let photobar = document.getElementsByClassName("css-1dbjc4n r-14lw9ot r-1tlfku8 r-t23y2h r-1phboty r-rs99b7 r-ku1wi2 r-1udh08x").item(0)
```

Comment:
I think this would break on every Twitter build/deploy because this is some autogenerated CSS classes.